truffle/src/main/ruby/core/rubinius/api/shims/mirror.rb

```diff
@@ -31,6 +30,7 @@ def self.module_mirror(obj)
         end
       end
     end
+    Truffle::Primitive.install_rubinius_primitive self, :module_mirror
```

Comment:
I'm not sure how to get `self` in a `@CoreMethod` anymore - `onSingleton` and `isModuleFunction` both set `needsSelf` to false.